src/chrome/chromeConnection.ts

```diff
@@ -155,7 +155,7 @@ export class ChromeConnection {
                 if (Array.isArray(responseArray)) {
                     // Filter out extension targets and other things
                     // Non-chrome scenarios don't always specify a type, so filter to include ones without a type at all
-                    let pages = responseArray.filter(target => target && (!target.type || target.type === 'page'));
+                    let pages = responseArray;//.filter(target => target && (!target.type || target.type === 'page'));
```

Comment:
Did a simple test. If I use the sample launch.json I posted in the PR, yes, sometimes(1 from 20 times running "launch" request) if chrome is already open, the debugger print that it picked up the first tab which probably will not be the right target to debug, however, if I add the `"url": "//nochadnphaoigfjnaclpecjelllocobm/dashboard"` to the launch.json, the problem is gone... Again, use `url`kill the problem...